src/components/layout-content/layout-content.tsx

```diff
@@ -0,0 +1,19 @@
+import {SideMenu} from '../side-menu/SideMenu'
+import {Layout} from 'antd'
+import React from 'react'
+
+const {Sider} = Layout
+
+export interface LayoutContentProps {
+  isSideMenuOpened?: boolean
+}
+
+export const LayoutContent: React.FC<LayoutContentProps> = ({isSideMenuOpened}) => {
+  return (
+    <Layout>
+      <Sider>
+        {isSideMenuOpened && <SideMenu entries={[{name: 'entry_1'}, {name: 'entry_2'}]}/>}
```

Comment:
I does not set the entries here, but passed from the parent. So, the basic interface of the component would be already usable

Reply:
My idea is to use in this component the service that will retrieve the menu configuration, and pass it to the SideMenu component